content/blog/2016/2016-01-27-jenkins-world-call-for-papers.adoc

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Jenkins World 2016: Call For Papers Is Open!"
+tags:
+- jenkins world
+- event
+- jenkins
+author: alyssat
+---
+
+image:/images/conferences/Jenkins-World_125x125.png[Jenkins World 2016,180,float="right",link="https://jenkins-cfp.herokuapp.com/events/jenkins-world-2016"]
+
+Planning is underway for the annual Jenkins Users Conference, to be held from
+*September 13th to 15th* in Santa Clara, California. Just like those before it,
+this year's event will feature many of the experts from the Jenkins community
+that help make Jenkins the most popular open source automation server on the
+planet. Unlike previous years however, we've found that we have outgrown the

```

Comment:
"*we* have outgrown"?  The previous JUCs were run by CloudBees, rather than the Jenkins community.